### PR TITLE
[FEATURE] HER-5128 Admin package

### PR DESCRIPTION
**Pull request by tiago-wd**
Change in the namespace and view files.

### CODE REVIEW

**Comment by tiago-wd** [comment]
@cyril-sf can you add Erick to github so he can also review?

File: src/AdminLTETemplatesServiceProvider.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace InfyOm\AdminLTETemplates;
+namespace InfyOm\Generator;
```
Comment (cyril-sf) [inline src/AdminLTETemplatesServiceProvider.php:3]: why this change?

**Review by cyril-sf — COMMENTED** [review]
@tiago-wd can you describe why you needed to change this library?

File: src/AdminLTETemplatesServiceProvider.php (hunk 1 above)
Comment (ericksrdev) [reply src/AdminLTETemplatesServiceProvider.php:3]: It is only a namespace change, but I do want to know the reason behind the change @tiago-wd

**Comment by ericksrdev** [comment]
Overall looks good for me, I need this PR merged in order to create a tag and replace the commit reference in our package manager

**Comment by ericksrdev** [comment]
@cyril-sf Can we merge this PR?

**Review by ericksrdev — APPROVED** [review]

File: src/AdminLTETemplatesServiceProvider.php (hunk 1 above)
Comment (tiago-wd) [reply src/AdminLTETemplatesServiceProvider.php:3]: To follow the same patter as the larave-generator package.

**Review by ericksrdev — APPROVED** [review]

File: src/AdminLTETemplatesServiceProvider.php (hunk 1 above)
Comment (ericksrdev) [reply src/AdminLTETemplatesServiceProvider.php:3]: @cyril-sf Is tiago's response answering your question or would you like a more described answer around the change?
